app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,46 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context="com.example.cocktaildictionary.MainActivity">
 
     <TextView
+        android:id="@+id/Id"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="Hello World!"
+        android:visibility="invisible"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <ProgressBar
+        android:id="@+id/progressBar"
+        style="?android:attr/progressBarStyle"
+        android:layout_width="0dp"
```

Comment:
By convention, we don't use hard coded values, e.g., 0dp, 50dp, 30sp etc.

Use Dimen resource file.

Reply:
I created a dimens.xml containing all my values